### PR TITLE
Finished the booking user journey

### PR DESCRIPTION
**Pull request by LRG-H**
Ben and I finished the booking user journey, please review.

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class BookingsController < ApplicationController
+  def show
```
Comment (euangt) [inline app/controllers/bookings_controller.rb]: No show method?

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    @booking.space = Space.find(params[:space_id])
+    if @booking.save
+      redirect_to booking_path(@booking)
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    @booking.destroy
```
Comment (euangt) [inline app/controllers/bookings_controller.rb:23]: You need to find the booking, no?

File: app/controllers/bookings_controller.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+class BookingsController < ApplicationController
+  def show
+    @bookings = Booking.all
```
Comment (ykookaasi) [inline app/controllers/bookings_controller.rb]: shall we call it index action
and you are showing all bookings, perhaps: 
@bookings = current_user.bookings

File: app/controllers/bookings_controller.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  def new
+    @booking = Booking.new
+  end
+
+  def create
+    @booking = Booking.new(bookings_params)
+    @booking.user = current_user
+    @booking.space = Space.find(params[:space_id])
+    if @booking.save
+      redirect_to booking_path(@booking)
```
Comment (ykookaasi) [inline app/controllers/bookings_controller.rb:15]: maybe redirect back to bookings_path?

File: app/controllers/bookings_controller.rb (hunk 2 above)
Comment (ykookaasi) [reply app/controllers/bookings_controller.rb:23]: Let's redirect it back to bookings_path?

File: app/controllers/bookings_controller.rb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+  def destroy
+    @booking = Booking.find(params[:id])
+    @booking.destroy
+  end
+
+  private
+
+  def bookings_params
+    params.require(:booking).permit(:user_id, :space_id, :start_date, :end_date)
```
Comment (ykookaasi) [inline app/controllers/bookings_controller.rb]: do we need permit ids?

**Review by ykookaasi — COMMENTED** [review]
some small stuff from me. check out the new action!

**Pull request merged**